mne/io/brainvision/brainvision.py

```diff
@@ -92,11 +92,12 @@ def __init__(self, vhdr_fname,
                 offsets = None
                 n_samples = n_samples // (dtype_bytes * n_data_ch)
 
+        orig_format = "single" if isinstance(fmt, dict) else fmt
         raw_extras = dict(
             offsets=offsets, fmt=fmt, order=order, n_samples=n_samples)
         super(RawBrainVision, self).__init__(
             info, last_samps=[n_samples - 1], filenames=[data_fname],
-            orig_format=fmt, preload=preload, verbose=verbose,
+            orig_format=orig_format, preload=preload, verbose=verbose,
             raw_extras=[raw_extras], orig_units=orig_units)
```

Comment:
This is safe because the `fmt` dict used for ASCII files is saved and passed in the `raw_extras` argument. I can't see a reason to duplicate it in `orig_format` which should contain a str. 
That said.. is it really `orig_format="single"`?

Reply:
Yes it is for ASCII, and this was my assumption as well, e.g. the test writes an ASCII file with 3 decimals only. This is why I've set it to `"single"`, but it does depend on what wrote the file and how it wrote it.

Reply:
This seems good enough then